tests/Doctrine/Tests/DBAL/Functional/Schema/OracleSchemaManagerTest.php

```diff
@@ -273,4 +275,90 @@ public function testListTableDateTypeColumns()
         $this->assertSame('datetime', $columns['col_datetime']->getType()->getName());
         $this->assertSame('datetimetz', $columns['col_datetimetz']->getType()->getName());
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createLargeNumberOfTables(): int
+    {
+        // Create a base table for test.
+        $sql = "CREATE TABLE tbl_test_2766_0 (x_id VARCHAR2(255) DEFAULT 'x' NOT NULL, x_data CLOB DEFAULT NULL NULL, x_number NUMBER(10) DEFAULT 0 NOT NULL, PRIMARY KEY(x_id))";
+        $this->_conn->executeUpdate($sql);
+
+        // Create a large number of tables with indexes and foreign keys.
+        for ($i = 1; $i < 649; $i++) {
```

Comment:
I don't see much value in having tests like this. Even if it performs slow for any reason, we won't see anything on CI as long as the test passes. It would make more sense to create a test which verifies that the number of executed SQL queries doesn't depend on the number of tables in the DB (if that's the case).